docs/en/integrations/data-ingestion/clickpipes/index.md

```diff
@@ -114,10 +102,10 @@ The supported formats are:
 
 | Format                                                                                    | Support     |
```

Comment:
@Ryado Would it make sense to expand this graph to show which formats are supported for which providers? For example, Avro will be supported for Kafka, but not S3. 
(Sorry, outside the scope of this PR but relevant to how this page becomes more generic).